src/testHelpers/loggerMock.js

```diff
@@ -0,0 +1,15 @@
+import nodeLogger from '../app/lib/logger.node';
+
+const mocks = {
+  error: jest.fn(),
+  warn: jest.fn(),
+  info: jest.fn(),
+  verbose: jest.fn(),
+  debug: jest.fn(),
+  silly: jest.fn(),
```

Comment:
silly?

Reply:
This is copied from existing parts of simorgh, `silly` is a supported logging type in the logger we use :)

Reply:
TIL. Thanks @dr3